source/includes/connect/mongoclient2.c

```diff
@@ -0,0 +1,48 @@
+#include <mongoc/mongoc.h>
+#include <bson/bson.h>
+
+int main(void) {
+
+  //start-connect-to-atlas
+  mongoc_client_t *client;
+  mongoc_server_api_t *api = NULL;
+  bson_t *command, reply;
+  bson_error_t error;
+  char *uri;
+  bool retval;
+  bool ok = true;
+
+  // Intialize the MongoDB C Driver
+  mongoc_init();
+
+  // Create a new client and connect to the server
+  uri = "mongodb+srv://<db_username>:<db_password>@<hostname/port>/?<options>";
```

Comment:
```suggestion
  uri = "mongodb+srv://<db_username>:<db_password>@<hostname:port>/?<options>";
```
The `/` within `<>` is confusing; suggest using `:` (for hostname and port) or `|` to denote choice.